cookbook/security/csrf_in_login_form.rst

```diff
@@ -33,7 +37,9 @@ provider available in the Security component:
                     # ...
                     form_login:
                         # ...
-                        csrf_provider: security.csrf.token_manager
+                        # Use csrf_provider in SF <2.8
```

Comment:
no need for such comment. this is why we have a versionned doc. Just put a ``versionadded`` section saying that the name has changed in 2.8